middleware/configuration/source/example/model.cpp

```diff
@@ -231,7 +231,7 @@ namespace casual
 
       groups:
          -  alias: A
-            note: "will get default queuebase: ${CASUAL_DOMAIN_HOME}/queue/groupA.gb"
+            note: "will get default queuebase: ${CASUAL_DOMAIN_HOME}/queue/groups/A.qb"
```

Comment:
Nice! You need to regenerate the documentation though. There's a _generator_ under bin that you should use. `--help` should be sufficient to understand.